tortoise/queryset.py

```diff
@@ -1325,10 +1326,15 @@ async def _execute(self) -> List[Any]:
         ]
         if self.flat:
             func = columns[0][1]
-            flatmap = lambda entry: func(entry["0"])  # noqa
+
```

Comment:
Why change this?

Reply:
It formatted by my editor custom setting, i will re-commit for remove these changes.